server/public/shared/i18n/i18n.go

```diff
@@ -78,6 +78,28 @@ func TranslationsPreInit(translationsDir string) error {
 	return initTranslationsWithDir(translationsDir)
 }
 
+// TranslationsPreInitFromBuffer loads translations from a buffer -- useful if
+// we need to initialize i18n from an embedded i18n file (e.g., from a CLI tool)
+func TranslationsPreInitFromBuffer(filename string, buf []byte) error {
```

Comment:
This is unfortunate, but the problem is: the CLI uses enterprise code which uses a couple translations to output feedback to the system console UI. So the feedback needs to be translated.
For now, the CLI only needs to support english, so this function is enough to translate only those couple of lines.
Eventually we're going to move the CLI's functionality into mmctl, and then we won't need this code anymore. I've added https://mattermost.atlassian.net/browse/MM-60972 to track removing this.